packages/schema/lib/utils/buildDocs.js

```diff
@@ -27,7 +27,9 @@ const walkSchemas = (InitSchema, callback) => {
 const collectSchemas = (InitSchema) => {
   const schemas = {};
   walkSchemas(InitSchema, (Schema) => {
-    schemas[Schema.id] = Schema;
+    if (!_.get(Schema, 'schema.docAnnotation.hide')) {
+      schemas[Schema.id] = Schema;
+    }
```

Comment:
@xavdid I feel like hiding entire schemas from the docs isn't supported, how do you feel about adding something like this?

Reply:
Nice! yeah, this would work.  

Your other (simpler) option is to not type the `firehoseWebhook` property at all, just make it a generic object like we do for `legacy`: 

https://github.com/zapier/zapier-platform/blob/6d4603b71fb3333a77d063729b8d7fe35583d20e/packages/schema/lib/schemas/AppSchema.js#L102-L109

That might simplify the schema portion of this PR in a significant way.

Reply:
ah interesting. So that means essentially you can put whatever you like in there, it's essentially the same as allowing additional properties in a schema? I think either way looks good, if you'd prefer all schemas are documented then I can switch it over to object to be consistent with the hidden legacy schema

Reply:
makes sense, we'd want to switch it over to a real documented schema if we ever decide to release to developers, but they may not ever happen - we may keep it private, or "on request"

Reply:
yeah, we'd document if if we released it. it's simple enough that hiding and simplifying it is ok for now. 

🚢  !